src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -59,7 +59,7 @@
         "symfony/scheduler": "^6.4.4|^7.0.4",
         "symfony/security-bundle": "^6.4|^7.0",
         "symfony/semaphore": "^6.4|^7.0",
-        "symfony/serializer": "^7.1",
+        "symfony/serializer": "^7.3",
```

Comment:
I would not bump the requirement here. We can instead remove the `serializer.normalizer.number` definition in the `FrameworkExtension` when the `NumberNormalizer` does not exist (you can take a look at how the `SnakeCaseToCamelCaseNameConverter` is handled inside `registerSerializerConfiguration()`).

Reply:
Ok, done :+1: Though I wonder if we should remove the definition also if `BcMath\Number` does not exist.

Reply:
No. In that case, your normalizer simply won't do anything, but that's fine.